packages/ember-htmlbars/tests/helpers/component_test.js

```diff
@@ -1,398 +0,0 @@
-import Controller from 'ember-runtime/controllers/controller';
-import { set } from 'ember-metal/property_set';
-import { get } from 'ember-metal/property_get';
-import run from 'ember-metal/run_loop';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-import ComponentLookup from 'ember-views/component_lookup';
-import EmberView from 'ember-views/views/view';
-import Component from 'ember-views/components/component';
-import compile from 'ember-template-compiler/system/compile';
-import computed from 'ember-metal/computed';
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var view, owner;
-
-QUnit.module('ember-htmlbars: {{#component}} helper', {
-  setup() {
-    owner = buildOwner();
-
-    owner.registerOptionsForType('template', { instantiate: false });
-    owner.register('component-lookup:main', ComponentLookup);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-    owner = view = null;
-  }
-});
-
-QUnit.test('component helper with bound properties are updating correctly in init of component', function() {
```

Comment:
moved to `@test component helper with bound properties are updating correctly in init of component`